js/search.js

```diff
@@ -914,17 +946,20 @@
 						}
 						if (moves.indexOf(moveid) >= 0) continue;
 						moves.push(moveid);
-						if (moveid === 'sketch') sketch = true;
+						if (moveid === 'sketch' && format !== 'balancedhackmons' && format !== 'bh') sketch = true;
```

Comment:
You know you can set a separate variable instead of repeating `format !== 'balancedhackmons' && format !== 'bh'` in like six places.

I actually strongly recommend it.

Reply:
`isBH`, put it on line 895.

Reply:
I had this originally and I'm not quite sure why I removed it